mobile/src/main/java/com/google/samples/apps/iosched/ui/sessiondetail/SessionDetailActivityViewModel.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.iosched.ui.sessiondetail
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.google.samples.apps.iosched.shared.result.Event
+
+class SessionDetailActivityViewModel : ViewModel() {
```

Comment:
You could alternatively have this class implement `ThemedActivityDelegate` so that the activity only needs one ViewModel (as in `MainActivityViewModel`). I have no strong opinion on it either way.

Reply:
Per your overall comment, by calling the method on the activity directly, there is no longer a need for these viewmodels. Marking as resolved since this is no longer relevant.